hack/local-up-karmada.sh

```diff
@@ -48,6 +49,16 @@ else
   echo "not pass"
   util::install_tools "sigs.k8s.io/kind" $kind_version
 fi
+
+echo -n "Preparing: 'kwokctl' existence check - "
+if util::cmd_exist kwokctl; then
+  echo "passed"
+else
+  echo "not pass"
```

Comment:
`no pass` means script run fail and will exit. Change to `kwokctl  is not found, and will install it...`

Reply:
Updated, But everything else seems to be echo 'not pass'